docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -1,30 +1,29 @@
 ---
-title: "$ - string interpolation - C# Reference"
+title: "$ - string interpolation - C# reference"
 ms.custom: seodec18
 description: String interpolation provides a more readable and convenient syntax to format string output than traditional string composite formatting.
-ms.date: 04/29/2019
+ms.date: 09/02/2019
 f1_keywords: 
   - "$_CSharpKeyword"
   - "$"
 helpviewer_keywords: 
   - "$ special character [C#]"
-  - "$ language element [C#]"
   - "string interpolation [C#]"
   - "interpolated string [C#]"
 author: pkulikov
 ms.author: ronpet
 ---
-# $ - string interpolation (C# Reference)
+# $ - string interpolation (C# reference)
 
-The `$` special character identifies a string literal as an *interpolated string*. An interpolated string is a string literal that might contain *interpolation expressions*. When an interpolated string is resolved to a result string, items with interpolation expressions are replaced by the string representations of the expression results. This feature is available in C# 6 and later versions of the language.
+The `$` special character identifies a string literal as an *interpolated string*. An interpolated string is a string literal that might contain *interpolation expressions*. When an interpolated string is resolved to a result string, items with interpolation expressions are replaced by the string representations of the expression results. This feature is available starting with C# 6.
```

Comment:
Nearly every where in the docs, I see language versions are mentioned with the `.0`. so I just changing `C# 6` with `C# 6.0`.

Reply:
`.0` indicates that there are or could be the point versions of the corresponding major version. C# 6 has no point versions, so `.0` suffix is not necessary